Lib/fontTools/ttLib/tables/otTables.py

```diff
@@ -202,17 +202,78 @@ def preWrite(self, font):
 
 		numShorts = 0
 		count = len(self.VarRegionIndex)
-		for item in self.Item:
-			for i in range(count - 1, numShorts - 1, -1):
-				if not (-128 <= item[i] <= 127):
-					numShorts = i + 1
-					break
-			if numShorts == count:
-				break
+		if hasattr(self, "Item") and self.Item:
+			for item in self.Item:
+				if item:
+					for i in range(count - 1, numShorts - 1, -1):
+						if not (-128 <= item[i] <= 127):
+							numShorts = i + 1
+							break
+					if numShorts == count:
+						break
 
 		rawTable['NumShorts'] = numShorts
 		return rawTable
 
+class GSUB(BaseTable):
```

Comment:
This is also not needed.  There's a cleaner way to support this.  I'm pushing that out in a few minutse.